planner/core/rule_join_elimination.go

```diff
@@ -91,7 +94,7 @@ func (o *outerJoinEliminator) isParentColsAllFromOuterTable(outerPlan LogicalPla
 	for _, col := range parentSchema.Columns {
 		columnName := &ast.ColumnName{Schema: col.DBName, Table: col.TblName, Name: col.ColName}
 		if c, err := outerPlan.Schema().FindColumn(columnName); c == nil {
-			log.Warn(err)
+			logutil.Logger(context.Background()).Warn("outerJoinEliminator raises an error", zap.Error(err))
```

Comment:
Why use `context.Background`? Why not just use `log.Warn`?

Reply:
After the discussion, we use context.Background() now.